tools/state-viewer/src/cli.rs

```diff
@@ -148,6 +152,26 @@ impl DumpStateRedisCmd {
     }
 }
 
+#[derive(Parser)]
+pub struct DumpTxCmd {
+    /// Specify which block to dump transactions for.
+    #[clap(long)]
+    height: BlockHeight,
```

Comment:
we might also want this to support block ranges -- but this can be done later.

Reply:
Acknowledged!